fail2ban/tests/files/logs/screensharingd

```diff
@@ -0,0 +1,18 @@
+# NOTE: dates here include years -- this is not the typical configuration for the system.log
+# file on Mac OS. However, without it the test routines will use 2004 as the year and matches will not pass.
+#
+# failJSON: { "match": false }
+Oct 27 2015 09:24:46 test1.beezwax.net screensharingd[1170]: Authentication: SUCCEEDED :: User Name: simon :: Viewer Address: 192.168.5.247 :: Type: DH
+# failJSON: { "match": false }
```

Comment:
Well, they are negative hits but indeed might be too far to be practically useful here?